plutus-playground-client/src/Action/View.purs

```diff
@@ -257,8 +262,12 @@ dragAndDropAction :: Int -> DragAndDropEventType -> DragEvent -> Maybe HAction
 dragAndDropAction index eventType = Just <<< ActionDragAndDrop index eventType
 
 onBigIntegerInput :: forall i r. (BigInteger -> i) -> IProp ( onInput :: Event, value :: String | r ) i
-onBigIntegerInput f = onValueInput $ map f <<< BigInteger.fromString
+onBigIntegerInput f = onValueInput $ map f <<< BigInteger.fromString <<< nonEmpty
+
+-- prevents the empty string from being given as input
+-- defaults to "1" because all the BigInteger fields here have a minimum value of 1
+nonEmpty :: String -> String
+nonEmpty str = if str == "" then "1" else str
```

Comment:
I don't like this way of doing it - encoding a number as a string to be turned back into a number. Yuck. This would be better:

```purescript
onBigIntegerInput f = onValueInput $ f <<< fromMaybe 1 <<< BigInteger.fromString <<< nonEmpty
```

...delete `nonEmpty` and move the comment up.

Reply:
Or `fromMaybe one` if you prefer.

Reply:
Good point, will fix. 👍